packages/ui/src/TabButton.tsx

```diff
@@ -24,13 +24,13 @@ const TabButton: FC<TabButtonProps> = ({ name, icon, active, type, showOnSm = fa
         onClick();
       }}
       className={clsx(
-        { 'text-brand bg-brand-100 bg-opacity-100 dark:bg-opacity-20': active },
-        'hover:bg-brand-100 flex items-center space-x-2 rounded-lg px-4 py-2 text-sm font-medium text-gray-500 hover:bg-opacity-100 dark:hover:bg-opacity-20 sm:px-3 sm:py-1.5'
+        { 'text-brand bg-blue-500 bg-opacity-100 dark:bg-opacity-20': active },
+        'text-brand flex items-center space-x-2 rounded-[2px] px-4 py-2 text-sm font-medium hover:bg-blue-500 hover:bg-opacity-100 dark:hover:bg-opacity-20 sm:px-3 sm:py-1.5'
```

Comment:
I'd go with this instead
```suggestion
{ 'bg-brand-500 text-dark dark:text-dark hover:cursor-default dark:bg-white': active },
        'text-dark dark:hover:text-dark hover:bg-brand-500 flex items-center space-x-2 rounded-full px-4 py-2 text-sm font-medium dark:text-white sm:px-3 sm:py-1.5'
```
<img width="715" alt="Capture d’écran 2023-05-25 à 14 02 19" src="https://github.com/ConsenSys/lineaster/assets/574787/1dc1be80-e93c-455e-875e-f02172767f4d">
<img width="734" alt="Capture d’écran 2023-05-25 à 14 02 12" src="https://github.com/ConsenSys/lineaster/assets/574787/844f0f7b-1d36-4e6b-b418-60e37800036f">